ethereum/statetransition/src/main/java/tech/pegasys/artemis/statetransition/util/EpochProcessorUtil.java

```diff
@@ -727,10 +731,12 @@ public static void finalUpdates(BeaconState state) {
     UnsignedLong index = next_epoch.plus(ENTRY_EXIT_DELAY).mod(LATEST_INDEX_ROOTS_LENGTH);
     // update hash tree root
     List<Bytes32> index_roots = state.getLatest_index_roots();
+    List<Validator> active_validators =
+        ValidatorsUtil.get_active_validators(
```

Comment:
Same thing I mentioned above

Reply:
rebased from master and now using the new `integerListHashTreeRoot`